### PR TITLE
feat(creator): add a cached fetch like for conventions calls

### PR DESCRIPTION
**Pull request by alexishecf**

### CODE REVIEW

File: packages/creator/README.md [hunk 1]
```diff
@@ -64,6 +64,8 @@ export type CreatorParams<T extends TransactionStrategy> = {
...
 - `fetchLike`: this is the fetch-like function used for making HTTP requests. It can be any function that has the same signature as the `fetch` function. By default, it uses the `defaultFetchLike` of `@arianee/utils`.
 - `protocolDetailsResolver`: you can provide a custom protocol details resolver, it can be any function that has the same signature as the `protocolDetailsResolver` function of `@arianee/arianee-protocol-client`.
+- `protocolDetailsResolver`: you can provide a custom protocol details resolver, it can be any function that has the same signature as the `protocolDetailsResolver` function of `@arianee/arianee-protocol-client`.
```
Comment (CyrilCartoux) [inline packages/creator/README.md:67]: doublon ?

**Review by CyrilCartoux — DISMISSED** [review]

File: packages/creator/src/lib/helpers/cachedFetchLike/cachedFetchLike.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+const defaultCache = new Map<string, string>();
+
+const isArianeeSchema = (url: string) =>
+  url.startsWith('https://cert.arianee.org/version');
```
Comment (mvaullerin) [inline packages/creator/src/lib/helpers/cachedFetchLike/cachedFetchLike.ts:4]: We should also cache https://cert.arianee.org/contractAddresses

Comment (alexishecf) [reply packages/creator/src/lib/helpers/cachedFetchLike/cachedFetchLike.ts:4]: yes adding it

**Comment by alexishecf** [comment]
closing this one as I did modifications in my other PR which result in a huge change of implementation so will be merged with it

**Pull request closed without merging**